### ISSUE TITLE
kitti experiments learning rate

### ISSUE THREAD

**Issue by brianhhu** [xingyizhou/CenterTrack#164]
Thanks for open sourcing your work. I noticed that for the kitti experiments, the default batch size/learning rate combination (bs=32, lr=1.25e-4) does not seem to be scaled according to the linear rule. The experiments use a batch size of 16, but the learning rate is kept the same. Do you know if this has any impact on the final trained model performance?
